docs/docsite/rst/community/collection_development_process.rst

```diff
@@ -39,9 +39,9 @@ Here's an overview of the PR lifecycle:
 Making your PR merge-worthy
 ===========================
 
-We do not merge every PR. Here are some tips for making your PR useful, attractive, and merge-worthy.
+We do not merge every PR. See :ref:`collection_quickstart` for  tips to make your PR useful, attractive, and merge-worthy.
```

Comment:
```suggestion
We do not merge every PR. See :ref:`collection_quickstart` for  tips to make your PR useful, attractive, and merge-worthy.
```
How about (also) adding this ref behind "Contributor opens a PR" in the paragraph above?
Later, when moving other docs, we could put the links to them in the corresponding items, for example, a ref to `review checklist` in "Developers, maintainers, community review the PR", etc. What do you think?

Reply:
sounds like a good plan.